3party/protobuf/CMakeLists.txt

```diff
@@ -0,0 +1,74 @@
+cmake_minimum_required(VERSION 3.2)
+
+project(protobuf C CXX)
+
+include_directories(. src ../../)
+
+add_compile_options(
+  "-Wall"
+  "-std=c++11"
+)
+
+if (NOT PLATFORM_WIN)
+  add_definitions(-DHAVE_PTHREAD)
```

Comment:
In general, this file will not work independently of our main project because we use the PLATFORM_WIN variable, so threads will have been found in the main project.